### ISSUE TITLE
htmx

### ISSUE THREAD

**Issue by jamestrew** [Waldeedle/todo#1]
![image](https://github.com/Waldeedle/todo/assets/66286082/8996ee97-f747-49c2-b425-820fece7297e)

where did you hear about this?

**Comment by Waldeedle** [comment]
I don't think you can avoid hearing about it at this point lol, I'm just playing around with HTMX while looking at other neat projects/tools in the ecosystem like Air

**Comment by jamestrew** [comment]
You're getting into neovim too? Lmao
Let me know if you need any help setting up. I've been using it for a few years plus I maintain a few popular plugins.

**Comment by Waldeedle** [comment]
Wait lol what gave that away? Yep I'm setting it up rn, but I still need to get more familiar with vim motions. The Normal mode doesn't really benefit me rn, I just go into insert lol.

**Comment by jamestrew** [comment]
I see you staring things on the GitHub home/activity page. Normal mode is king. I started with vim in vs code with the vim extension. Just get used to editing without using a mouse or arrow keys.
